ui/intermittent-failures/GraphAlternateView.jsx

```diff
@@ -0,0 +1,87 @@
+/* eslint-disable array-callback-return */
+
+import React from 'react';
+import ReactTable from 'react-table';
+
+const GraphAlternateView = ({ graphData, colNum }) => {
+  const columnsOne = [
+    {
+      Header: 'Date',
+      accessor: 'x',
+    },
+    {
+      Header: 'Failure Count per Push',
+      accessor: 'y',
+    },
+  ];
+
+  const columnsTwo = [
+    {
+      Header: 'Date',
+      accessor: 'x',
+    },
+    {
+      Header: 'Failure Count',
+      accessor: 'y',
+    },
+    {
+      Header: 'Push Count',
+      accessor: 'z',
+    },
+  ];
+
+  const alternateGraph = {
+    column: [],
+    data: [],
+  };
+
+  const convertDateToString = date =>
+    date.toLocaleDateString('en-US', {
+      day: 'numeric',
+      month: 'short',
+    });
+
+  graphData.map((item, index) => {
+    if (colNum === 1) {
+      item.data.map(itemOne => {
```

Comment:
Since you're not returning anything in map, you can use a `for` loop instead.

Reply:
...or use `forEach()`